src/test/components/TrackContextMenu.test.js

```diff
@@ -721,8 +722,8 @@ describe('timeline/TrackContextMenu', function () {
   });
 
   describe('when a local track is right clicked', function () {
-    function setupLocalTrack() {
-      const results = setup();
+    function setupLocalTrack(profile) {
+      const results = setup(profile);
```

Comment:
After all I don't use this change, but still kept it because this does no harm...